tools/codegen/model.py

```diff
@@ -606,6 +606,11 @@ class BackendIndex:
     # Other backend-specific information that is on a per-operator basis
     index: Dict['OperatorName', BackendMetadata]
 
+    # Logging macros that are inserted into wrappers. Only really used by external backends.
+    per_op_log: Optional[str] = None
```

Comment:
nit: renaming these to `per_op_setup/per_argument_setup/cpu_fallback_setup` to make these fields more general, wdyt?

Reply:
Actually, I'm planning on removing these from the PR (but let me know if you agree). From Jack's [comment](https://github.com/pytorch/xla/pull/2944#discussion_r635540480), it sounds fine to remove the custom logging from the codegen, which is the only reason I added it in.